telemetry/data_feature_usage.go

```diff
@@ -121,20 +122,27 @@ type TableClusteredInfo struct {
 	// NA means this field is no meaningful information
 }
 
+// NewClusterIndexUsage records the clustered index usage info of all the tables.
+type NewClusterIndexUsage struct {
+	// The number of user's tables with clustered index enabled.
+	NumClusteredTables uint64 `json:"numClusteredTables"`
+	// The number of user's tables.
+	NumTotalTables uint64 `json:"numTotalTables"`
+}
+
 // getClusterIndexUsageInfo gets the ClusterIndex usage information. It's exported for future test.
-func getClusterIndexUsageInfo(ctx sessionctx.Context) (cu *ClusterIndexUsage, err error) {
+func getClusterIndexUsageInfo(ctx sessionctx.Context) (ncu *NewClusterIndexUsage, cu *ClusterIndexUsage, err error) {
+	var newUsage NewClusterIndexUsage
 	usage := make(ClusterIndexUsage)
```

Comment:
Please change `usage` to a global variable(a constant actually), so that we don't need to `make` it every time.